getting-started/android-with-appengine/README.md

```diff
@@ -11,13 +11,12 @@ The sample is called **Emojify**.
 
 Example of source and emojified images:
 
-<image src="screenshots/meetup.jpg" width="430px"/> <<image src="screenshots/emojified-meetup.jpg" width="430px"/>
+<image src="screenshots/meetup.jpg" width="430px"/> <image src="screenshots/emojified-meetup.jpg" width="430px"/>
 Picture taken at [DroidCon NYC Extended](https://dcnyc-extended-2018.splashthat.com/).
-            
-### Cloud Client Libraries
-The app integrates two Google Cloud Client Libraries:
-* [Google Cloud Storage](https://cloud.google.com/storage): hosts input and emojified images
-* [Google Cloud Vision](https://cloud.google.com/vision): used to perform face detection and image annotation on input image
+
+# Tutorial
+
+**This [codelab](https://g.co/codelabs/emojify) walks you through how to build and deploy Emojify.**
 
 ### Emojify workflow:
```

Comment:
I don't really feel like we need this section, since these parts are also described above, below, and especially in the codelab.

Reply:
This is what I did for my codelab READMEs, you can see how little info I have there: https://github.com/GoogleCloudPlatform/kotlin-samples/blob/master/appengine/springboot/README.md